### DISCUSSION TITLE
View New Flashcards Earlier

### DISCUSSION THREAD

**Discussion by ryanwwest** [st3v3nmw/obsidian-spaced-repetition#457] [Q&A]
Not exactly the same thing, but is there a way to review newly added flashcards before going through the entire backlog of previously viewed cards that are due to be viewed again? I have like 250 cards that I have to view before seeing any of the new 5 ones I added, so I rarely get to reviewing the newest information.

**Comment by st3v3nmw** [comment]
Hi @ryanwwest,
This is currently not supported. We're tracking the issue in #500
